BrowserKit/Sources/ComponentLibrary/Buttons/LinkButtonViewModel.swift

```diff
@@ -23,9 +23,9 @@ public struct LinkButtonViewModel {
                 a11yIdentifier: String,
                 font: UIFont = FXFontStyles.Regular.callout.scaledFont(),
                 contentInsets: NSDirectionalEdgeInsets = NSDirectionalEdgeInsets(top: UX.verticalInset,
-                                                                                 leading: UX.horizontalInset,
+                                                                                 leading: 0,
```

Comment:
it looks like we're updating this for all our link buttons, but is only needed for this case. What do you think about passing this in when we initialize the `LinkButtonViewModel` for tracking protection?